app/views/includes/iFormInProgressBanner.scala.html

```diff
@@ -21,10 +21,24 @@
 
 @viewModel.durationMessage.map { durationMessage =>
 
-    <div id="isAnyFormInProgressBanner" class="tai-progress-panel">
-        <h2 class="heading-small flush--top">@messages("tai.whatDoYouWantToDo.iformPanel.p1")</h2>
-        <p>@durationMessage</p>
-        <p><a id="checkProgressLink" href="@appConfig.checkUpdateProgressLinkUrl">@messages("checkProgress.link")</a></p>
-    </div>
+<!--  <div id="isAnyFormInProgressBanner" class="tai-progress-panel">-->
```

Comment:
commented out code still there

Reply:
Thanks. Forgot to remove it after testing 👍